src/docs/index.md

```diff
@@ -35,13 +36,16 @@ Some of the new content includes:
 * Deep dive on Flutter internals,
   [Inside Flutter](/docs/resources/inside-flutter)
 * [Technical videos](/docs/resources/videos) page
-* How to connect [a native debugger _and_
-  a Dart debugger to your app](/docs/testing/oem-debuggers)
-  (iOS info in process)
+* [State management](/docs/development/data-and-backend/state-mgmt) landing page
```

Comment:
In this section of the page (describing new content), could we drop the qualifiers like "landing page" or even just "page"? /cc @fertolg